deeplearning4j-scaleout/spark/dl4j-spark-ml/src/main/spark-1/org/deeplearning4j/spark/ml/impl/SparkDl4jNetwork.scala

```diff
@@ -0,0 +1,32 @@
+package org.deeplearning4j.spark.ml.impl
+
+import org.apache.spark.ml.param.ParamMap
+import org.apache.spark.ml.util._
+import org.apache.spark.mllib.linalg.Vector
+import org.apache.spark.mllib.regression.LabeledPoint
+import org.apache.spark.sql.{DataFrame, Row}
+import org.deeplearning4j.spark.impl.multilayer.SparkDl4jMultiLayer
+import org.deeplearning4j.spark.ml.utils.DatasetFacade
+
+
+final class SparkDl4jNetwork(override val uid: String) extends SparkDl4jNetworkWrapper[Vector, SparkDl4jNetwork, SparkDl4jModel](uid) {
```

Comment:
I can change it, but it might be weird, since this is the class that a user would implement. I was trying to go for a similar naming convention on what dl4j has for the mllib network.